src/main/groovy/net/rdrei/android/buildtimetracker/reporters/NetworkReporter.groovy

```diff
@@ -7,13 +7,18 @@ class NetworkReporter extends AbstractBuildTimeTrackerReporter {
 
     private HttpClient httpClient
 
+    private def isJenkinsJob
+    private def version
+
     NetworkReporter(
             Map<String, String> options,
             Logger logger,
             HttpClient httpClient
     ) {
         super(options, logger)
         this.httpClient = httpClient
+        this.isJenkinsJob = getOption("is_jenkins_job", false)
+        this.version = getOption("version", 1)
```

Comment:
I'm not sure if it makes sense to specify version as option. I'd say it should be hardcoded and correspond to plugin version.

Reply:
Good point!